application/models/Adif_data.php

```diff
@@ -93,7 +93,10 @@ function export_custom($from, $to, $station_id, $exportLotw = false) {
             $this->db->where("date(".$this->config->item('table_name').".COL_TIME_ON) <= '".$to."'");
         }
         if ($exportLotw) {
+            $this->db-group_start();
```

Comment:
Typo, `>` is missing, should be: `$this->db->group_start();`

Reply:
Foxed